src/sinks/splunk_hec/common/acknowledgements.rs

```diff
@@ -201,7 +201,15 @@ impl HecAckClient {
             .freeze();
         let request = self
             .http_request_builder
-            .build_request(request_body_bytes, "/services/collector/ack", None)
+            .build_request(
+                request_body_bytes,
+                "/services/collector/ack",
+                None,
+                None,
+                None,
+                None,
+                None,
+            )
```

Comment:
I feel like this would be more readable if these were named fields - as I've got no idea what any of those `None`s are 🙂

Reply:
Clippy agrees.